jit-application/src/main/java/org/dcsa/jit/controller/TimestampController.java

```diff
@@ -38,6 +41,7 @@ public void create(@Valid @RequestBody TimestampTO timestamp) {
         + invalid.DCSAResponsibleAgencyCode() + "\" (\""
         + invalid.DCSAResponsibleAgencyCode().getLegacyAgencyCode() + "\")");
     }
-    timestampService.createAndRouteMessage(timestamp);
+    OperationsEvent operationsEvent = timestampService.createAndRouteMessage(timestamp);
+    timestampNotificationMailService.enqueueEmailNotificationForEvent(operationsEvent);
```

Comment:
We are losing the `@Transactional` protection here (as it is on the service layer).  This opens a race-condition between creating the operations but before enqueuing the email notification in which we can lose the email notification in a crash.